packages/oats/src/generate-types.ts

```diff
@@ -938,32 +938,15 @@ export function run(options: Options) {
           ts.createPropertyAssignment('maximum', ts.createNumericLiteral(schema.maximum + ''))
         );
       }
-      return ts.createObjectLiteral(properties, true);
-    }
-    if (schema.type === 'integer') {
-      const enumValues = schema.enum
-        ? [
-            ts.createPropertyAssignment(
-              'enum',
-              ts.createArrayLiteral(schema.enum.map(i => ts.createNumericLiteral('' + i)))
-            )
-          ]
-        : [];
-      const properties = [
-        ts.createPropertyAssignment('type', ts.createStringLiteral('integer')),
-        ...enumValues
-      ];
-      if (schema.minimum != null) {
+      if (schema.convert != null) {
```

Comment:
from where does this `convert` prop come from? I can't find it from the openapi 3.0 spec

Reply:
removed! (P.S. there's actually a way to extend an open api options by using `x-` prefix, but i agree it's better not overuse it)
Anyways the only use case is parsing request parameters, and we want always to do this if required by schema type